lib/utils.js

```diff
@@ -114,12 +109,60 @@ const logResultErrors = result => {
   }
 }
 
+const getTSBabelOptions = () => {
+  const ts = getTsInstance()
+  const { compilerOptions } = getTsConfig()
+  const TSBabelOptions = {
+    presets: [
+      '@babel/env',
+      ['@babel/preset-typescript', { allExtensions: true }]
+    ],
+    plugins: [
+      ['@babel/plugin-proposal-decorators', { legacy: true }],
+      ['@babel/plugin-proposal-class-properties', { loose: true }],
+      '@babel/plugin-proposal-object-rest-spread',
+      '@babel/plugin-syntax-dynamic-import',
+      ['@babel/plugin-transform-runtime', { corejs: 2 }]
```

Comment:
I can see why we're adding `@babel/plugin-proposal-decorators`, but why are we adding '@babel/plugin-proposal-object-rest-spread' and '@babel/plugin-transform-runtime'?